stackslib/Cargo.toml

```diff
@@ -93,6 +93,7 @@ features = ["serde"]
 [dependencies.time]
 version = "0.2.23"
 features = ["std"]
+mutants = "0.0.3"
```

Comment:
This should either be a dev-dependency, or an optional and off-by-default dependency

Reply:
moved it to dev-dependency